frame/treasury/src/lib.rs

```diff
@@ -106,59 +52,41 @@
 //! - `reject_proposal` - Reject a proposal, slashing the deposit.
 //! - `approve_proposal` - Accept the proposal, returning the deposit.
 //!
-//! Tipping protocol:
-//! - `report_awesome` - Report something worthy of a tip and register for a finders fee.
-//! - `retract_tip` - Retract a previous (finders fee registered) report.
-//! - `tip_new` - Report an item worthy of a tip and declare a specific amount to tip.
-//! - `tip` - Declare or redeclare an amount to tip for a particular reason.
-//! - `close_tip` - Close and pay out a tip.
-//!
-//! Bounty protocol:
-//! - `propose_bounty` - Propose a specific treasury amount to be earmarked for a predefined set of
-//! tasks and stake the required deposit.
-//! - `approve_bounty` - Accept a specific treasury amount to be earmarked for a predefined body of work.
-//! - `propose_curator` - Assign an account to a bounty as candidate curator.
-//! - `accept_curator` - Accept a bounty assignment from the Council, setting a curator deposit.
-//! - `extend_bounty_expiry` - Extend the expiry block number of the bounty and stay active.
-//! - `award_bounty` - Close and pay out the specified amount for the completed work.
-//! - `claim_bounty` - Claim a specific bounty amount from the Payout Address.
-//! - `unassign_curator` - Unassign an accepted curator from a specific earmark.
-//! - `close_bounty` - Cancel the earmark for a specific treasury amount and close the bounty.
-//!
-//!
 //! ## GenesisConfig
 //!
 //! The Treasury module depends on the [`GenesisConfig`](./struct.GenesisConfig.html).
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(test)]
 mod tests;
+
```

Comment:
no reason for newlines here imo

Reply:
:-) removed it, Could you plz point me to any Linting tool or coding guideline docs, helpful to avoid such stuffs in future ...

Reply:
Unfortunately we do not use a linter. We just have a style guide... i dont have much useful advice here, sorry :(